pbft/src/pbft-core/transportation.go

```diff
@@ -29,7 +29,7 @@ import (
 func MakeTransportations(index int) []*rpc.Client {
 	// index: the index if the server itself
 	clientList := make([]*rpc.Client, 0)
-	lst, ports := GetIPConfigs("ipconfig")
+	lst, ports, _ := GetIPConfigs("ipconfig")
```

Comment:
do we need grpcports to be returned from GetIPConfigs ? because I see another list GrpcPorts []int defined here somewhere..

Reply:
Right now MakeTransportations() isn't called anywhere. Hence, just left it untouched except what was required to make sure compilation succeeds. We do not need the grpc ports to be returned here for now.